docs/docs/getting-started/glossary.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Glossary
+sidebar_label: Glossary
+sidebar_position: 3
+---
+
+This glossary provides concise definitions for some of the key concepts within Flagsmith:
+
+- [**A/B Testing**](/advanced-use/ab-testing.md): A method of testing different feature variants with different user groups, often implemented using multivariate flags and percentage splits.
+
+- [**Core API**](/edge-api/Overview.md#core-api): Flagsmith's private API for programmatic control of the platform.
+
+- [**Edge API / Edge Proxy**](/advanced-use/edge-api.md): Flagsmith's publicly accessible API, specifically intended for use with our SDKs.
```

Comment:
The Edge API and Edge Proxy are 2 completely separate concepts, let's not group them here.